src/Datadog.Trace.ClrProfiler.Native/CorProfiler.cpp

```diff
@@ -52,6 +53,15 @@ HRESULT STDMETHODCALLTYPE CorProfiler::Initialize(IUnknown* pICorProfilerInfoUnk
     }
     */
 
+    WCHAR integration_file_path[MAX_PATH]{};
+    const DWORD integration_file_path_length = GetEnvironmentVariable(L"DATADOG_INTEGRATIONS", integration_file_path, _countof(integration_file_path));
```

Comment:
Not required for this PR, but later we should allow loading multiple of these integration files:
- built-in integrations (deployed with the installer to `C:\Program Files\...`, not editable by user)
- machine-wide custom integrations
- application-local custom integrations

Related: we should think about how this will relate to configuration files. Will they be completely separate? Or are integrations just a node within a top-level configuration node? Will the path to the integration file be defined in the configuration file?